searx/locales.py

```diff
@@ -10,8 +16,15 @@
     "oc": "Lenga D'òc (Occitan)",
     "nl_BE": "Vlaams (Dutch, Belgium)",
 }
+"""Mapping of locales and their description.  Locales e.g. 'fr' or 'pt_BR'
+(delimiter is *underline* '_')"""
+
 UI_LOCALE_CODES: List[str] = []
+"""List of locales e.g. 'fr' or 'pt-BR' (delimiter is '-')"""
+
 RTL_LOCALES: Set[str] = set()
+"""List of *Right-To-Left* locales e.g. 'he' or 'fa_IR' (delimiter is
+*minus* '-')"""
```

Comment:
`delimiter is *minus* '-'` --> `delimiter is *minus* '_'`

Reply:
Ooops .. fixed.